main.py

```diff
@@ -1,25 +1,35 @@
 from forum_service import ForumService
-from post_cleaner import PostCleaner
 from db_manager import DBManager
+from Models.forum_post_model import ForumPostModel
 
+#initialize our classes
 db = DBManager()
+fs = ForumService()
```

Comment:
I still hate the non-descriptive variable names

Reply:
too bad

Reply:
we can change them later